src/components/ViolationForm.js

```diff
@@ -0,0 +1,250 @@
+  const [towns, setTowns] = useState([]);
+
+  const api_endpoint = 'http://localhost:4000';
+
+  useEffect(() => {
+    console.log(selectedElectionRegion);
+    axios
+      .get(`${api_endpoint}/election_regions`)
+      .then((res) => setElectionRegions(res.data));
+  }, []);
+
+  useEffect(() => {
+    console.log('in 2nd use effect before if');
+    if (selectedElectionRegion != '') {
+      console.log('get municipalities');
+      getMunicipalities(selectedElectionRegion);
+    }
+    if (selectedElectionRegion != '' && selectedMunicipality != '') {
+      console.log('in 2nd use effect after if');
+      axios
+        .get(
+          `${api_endpoint}/towns?country=00&election_region=${selectedElectionRegion}&municipality=${selectedMunicipality}`
+        )
+        .then((res) => setTowns(res.data));
+    }
+  }, [selectedElectionRegion, selectedMunicipality]);
+  const getElectionRegions = () => {
+    return electionRegions.map((election_region) => {
+      return (
+        <option key={election_region.code} value={election_region.code}>
+          {election_region.name}
+        </option>
+      );
+    });
+  };
+  const getMunicipalities = (selectedElectionRegion) => {
+    console.log('here');
+
+    const filteredRegions = electionRegions.filter(
+      (electionRegion) => electionRegion.code == selectedElectionRegion
+    );
+    const municipalities = [];
+    filteredRegions[0].municipalities.forEach((municipality) => {
+      municipalities.push(municipality);
+    });
+
+    return municipalities.map((municipality) => {
+      return (
+        <option key={municipality.code} value={municipality.code}>
+          {municipality.name}
+        </option>
+      );
+    });
+  };
+  const createTownOptions = () => {
+    console.log('in create town');
+    return towns.map((town) => {
+      return (
+        <option id={town.code} key={town.code} value={town.code}>
+          {town.name}
+        </option>
+      );
+    });
+  };
+  const onSubmit = (data) => {
+    const body = {
+      description: data.description,
+      town: 68134,
+    };
+    console.log(JSON.stringify(body));
+    axios
+      .post(`${api_endpoint}/violations`, body, {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      })
+      .then((res) => console.log(res.data))
+      .catch((err) => console.log(err));
+  };
+
+  return (
+    <CommentFormStyle onSubmit={handleSubmit(onSubmit)}>
+      <div className="form-control">
+        <label>Секция в:</label>
+        <div className="radio_button_1">
+          <input
+            type="radio"
+            id="fieldBg"
+            {...register('fieldBg', { required: false })}
+          />
+          <label className="radioLabel">България</label>
+          <input
+            type="radio"
+            id="fieldForeign"
+            {...register('fieldForeign', { required: false })}
+          />
+          <label className="radioLabel">Чужбина</label>
```

Comment:
Let's pre-select Bulgaria by default and list election regions from Bulgaria by default. If person selects Abroad, election regions and municipalities should be hidden.

Exactly like how the mobile app behaves.